integrations/pjrt/README.md

```diff
@@ -37,6 +37,14 @@ pip install -v --no-deps -e python_packages/iree_cpu_plugin
 JAX_PLATFORMS=iree_cpu python -c "import jax; a = jax.numpy.asarray([1, 2, 3, 4, 5, 6, 7, 8, 9]); print(a + a);"
 ```
 
+To pass additional compile options to IREE, you can use environment variable `IREE_PJRT_IREE_COMPILER_OPTIONS`.
+
+For example:
+```shell
+export IREE_PJRT_IREE_COMPILER_OPTIONS=--iree-scheduling-dump-statistics-format=csv
+JAX_PLATFORMS=iree_cpu python -c "import jax; a = jax.numpy.asarray([1, 2, 3, 4, 5, 6, 7, 8, 9]); print(a + a);"
+```
```

Comment:
Thanks for the concrete example here. Can you move it under a new heading so the "Verify basic functionality" heading is kept minimal?

````suggestion
## Advanced settings

To pass additional compile options to IREE during JIT compilation, you can use
the `IREE_PJRT_IREE_COMPILER_OPTIONS` environment variable. This variable can
be set to a space-delimited list of flags that would be passed to the
`iree-compile` command-line tool.

For example:
```shell
export IREE_PJRT_IREE_COMPILER_OPTIONS=--iree-scheduling-dump-statistics-format=csv
JAX_PLATFORMS=iree_cpu python -c "import jax; a = jax.numpy.asarray([1, 2, 3, 4, 5, 6, 7, 8, 9]); print(a + a);"
```
````


Might also want to give a more complicated example with multiple options, quotes, and spaces, to show how the parsing works.

Reply:
Good suggestion. Sound clearer to me now.